.devcontainer/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM centos/devtoolset-6-toolchain-centos7:latest
+
+LABEL \
+  description="Externpro Centos 7 Development Container" \
+  name="externpro-dev"
+
+USER 0
+RUN set -eux; \
+  yum --assumeyes --quiet update; \
+  yum install --assumeyes \
+    epel-release \
+    # This repository gives us more modern versions of some software, such as Git.
+    https://packages.endpoint.com/rhel/7/os/x86_64/endpoint-repo-1.7-1.x86_64.rpm; \
+  yum install --assumeyes \
+    cmake3 \
+    git \
```

Comment:
in the past this has installed git 1.8. terrible.

Reply:
consider installing `rh-git218`

Reply:
See the comment on line 12. This installs git 2.24.1. I can check for a more official RH package, but my container can't access yum repositories at the moment. I can double check later.

Reply:
I can't get Git to be available if I install rh-git218 packages. I'd prefer to stick with epel if we could, but whichever rh-git218 package I try, the git command is not available.